base/ca/src/main/java/com/netscape/ca/CAService.java

```diff
@@ -1628,7 +1631,7 @@ public boolean service(IRequest request) throws EBaseException {
 
     public serviceCheckChallenge(CAService service) {
         try {
-            mSHADigest = MessageDigest.getInstance("SHA1");
+            mSHADigest = MessageDigest.getInstance("SHA-256");
```

Comment:
I think we need @ladycfu or @jmagne to confirm if this is OK. We need to know how to test it too.

Reply:
Absolutely.